exp_build/experiment.js

```diff
@@ -192,6 +196,15 @@ class Experiment {
       bp: Config.build_path(this.src) + "/",
     };
 
+    if(options.codeditor) {
```

Comment:
This processing is not expected to happen here. This is supposed to happen in plugin.js in a generic manner.

Reply:
resolved

Reply:
Sir how to make the processing generic? The plugins with defined scope are handle together whereas these component based plugins have their own build option. Hence we will exclusiveyly have to check for their option and then proceed further

Reply:
I havent made this into generic as of now